dove/src/index/mod.rs

```diff
@@ -265,16 +256,129 @@ impl Index {
         self.deps_roots
             .iter()
             .map(PathBuf::from)
-            .filter(|path| {
-                let com = path
-                    .components()
-                    .filter_map(|el| el.as_os_str().to_str())
-                    .collect::<Vec<&str>>();
-                com.iter()
-                    .find_position(|el| el == &&".external")
-                    .and_then(|(pos, _)| com.get(pos + 1))
-                    .map_or(false, |el| el.starts_with("git_"))
-            })
+            .filter(|path| is_external_git(path))
             .for_each(remove);
     }
+
+    /// difference in indexes
+    pub fn diff<'a>(&'a self, index: &'a Index) -> Vec<Diff<'a>> {
+        let cur_paths = &self.deps_roots;
+        let oth_paths = &index.deps_roots;
```

Comment:
Same here, `ours` and `theirs` are shorter and more accurate than `cur_paths` and `oth_paths`

Reply:
Thanks. Fixed